### DISCUSSION TITLE
Pull Request Labeler

### DISCUSSION THREAD

**Discussion by sandreenko** [community/community#25843] [Actions]
<p>Hello, I want to use new actions to automatically label my PRs as described here:</p>
<p><a href="https://github.com/actions/labeler/blob/master/README.md" rel="nofollow noopener">https://github.com/actions/labeler/blob/master/README.md</a></p>
<p>So I added <a href="https://github.com/sandreenko/coreclr/blob/master/.github/labeler.yml" rel="nofollow noopener">https://github.com/sandreenko/coreclr/blob/master/.github/labeler.yml</a> file with:</p>
<p>area-CodeGen:</p>
<ul>
<li>src/jit/**/*</li>
</ul>
<p>that in my understanding should apply <code>area-CodeGen</code> to all PRs that have changes in <code>src/jit/</code> folder (including subfolders), </p>
<p>and I created a simple PR to test that: <a href="https://github.com/sandreenko/coreclr/pull/17" rel="nofollow noopener">https://github.com/sandreenko/coreclr/pull/17</a></p>
<p>but no label was applied, could somebody tell me what I am doing wrong?</p>

**Comment by cschleiden** [comment]
<p>Your workflow needs to be in the <code>.github/workflows</code> folder, see <a href="https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file" rel="nofollow noopener">https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file</a> for creating a new workflow.</p>
<p>Thanks,</p>
<p> Chris</p>

**Comment by sandreenko** [comment]
<p>Hi Chris, thanks for your reply.</p>
<p>Does it mean that <a href="https://github.com/actions/labeler/blob/master/README.md" rel="nofollow noopener">https://github.com/actions/labeler/blob/master/README.md</a> this part(Basic Examples and Common Examples) is not working? </p>
<p>I did not want to utilize any machines for that.</p>

**Comment by oldskool** [comment, marked as answer]
<blockquote>
<p><a class="mention" href="/sandreenko">@sandreenko</a> wrote:</p>
</blockquote>
<blockquote>
<p>Does it mean that <a href="https://github.com/actions/labeler/blob/master/README.md" rel="nofollow noopener">https://github.com/actions/labeler/blob/master/README.md</a> this part(Basic Examples and Common Examples) is not working? </p>
</blockquote>
<p>The examples are fine, they configure the action as explained. But, you still need to add an actual workflow to your project, as described in the <a href="https://github.com/actions/labeler/blob/master/README.md#create-workflow" rel="nofollow noopener">second half of the README</a>. Without it, GitHub does not automagically start running the action, you need to explicitly enable the workflow.</p>

**Comment by ibakshay** [comment]
<p>The Labeller won’t work for any Pull Requests coming from the forked repository and this issue is discussed <a href="https://github.com/actions/labeler/issues/12" rel="nofollow noopener">here</a>.</p>
